lib/internal/modules/cjs/loader.js

```diff
@@ -198,6 +199,8 @@ const onRequire = getLazy(() => tracingChannel('module.require'));
 
 const relativeResolveCache = { __proto__: null };
 
+const ci = require('code_integrity');
```

Comment:
I would put this into an internal module right now.

Reply:
Just to make sure - this just involves moving it into the lib/internal/ folder? Let me know if I need to do anything else!